lib/src/components/NxModal/NxModal.scss

```diff
@@ -29,7 +29,6 @@
     flex-basis: auto;
     flex-direction: column;
     flex-shrink: 1;
-    overflow-y: hidden;
```

Comment:
Removing this or setting x to hidden as well fixes it, but not sure if it that's a good solution.
`overflow: hidden`

Reply:
I find it hard to believe that this change doesn't cause problems with scrolling form modals (I can't check of course as I have no access to Safari 16.1).  At the same time, the other solution you propose would seem like it shouldn't cause any problems especially in light of this disclaimer which `NxModal` has always carried:
![image](https://user-images.githubusercontent.com/3630091/210634192-6b140c8c-acfc-44ea-ac34-214856c7f486.png)

So I'd go with the other solution.

Reply:
> I find it hard to believe that this change doesn't cause problems with scrolling modals (I can't check of course as I have no access to Safari 16.1). At the same time, the other solution you propose would seem like it shouldn't cause any problems especially in light of this disclaimer which `NxModal` has always carried: ![image](https://user-images.githubusercontent.com/3630091/210634192-6b140c8c-acfc-44ea-ac34-214856c7f486.png)
> 
> So I'd go with the other solution.

Ah yeah I forgot to update the comment, but I went with the other solution because of that issue here:
14a2e69a4d0ae50738a80bfa0506ce0deb4ac164